docs/pipeline.rst

```diff
@@ -478,11 +499,15 @@ intent_classifier_tensorflow_embedding
           # flag if to tokenize intents
           "intent_tokenization_flag": false
           "intent_split_symbol": "_"
+          # visualization of accuracy
+          "calc_acc_ones_in_epochs": 10  # small values affect performance
```

Comment:
'affect performance'? a positive or negative effect?

Reply:
negative

Reply:
you go through training data and predict more often